docs/source/known-issues.rst

```diff
@@ -23,6 +23,49 @@ HIP
+      2572 |   static_assert(is_device_copyable_v<FieldT> ||
+           |                 ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+      2573 |                     detail::IsDeprecatedDeviceCopyable<FieldT>::value,
+           |                     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+      /usr/bin/compiler/../../include/sycl/types.hpp:2605:7: note: in instantiation of template class 'sycl::detail::CheckFieldsAreDeviceCopyable<(lambda at /usr/include/oneapi/dpl/pstl/hetero/dpcpp/parallel_backend_sycl.h:1578:83), 4>' requested here
+      2605 |     : CheckFieldsAreDeviceCopyable<FuncT, __builtin_num_fields(FuncT)>,
+           |       ^
+     /usr/bin/compiler/../../include/sycl/types.hpp:2613:7: note: in instantiation of template class 'sycl::detail::CheckDeviceCopyable<(lambda at /usr/include/oneapi/dpl/pstl/hetero/dpcpp/parallel_backend_sycl.h:1578:83)>' requested here
+      2613 |     : CheckDeviceCopyable<KernelType> {};
+           |       ^
+     /usr/bin/compiler/../../include/sycl/handler.hpp:1652:5: note: in instantiation of template class 'sycl::detail::CheckDeviceCopyable<sycl::detail::RoundedRangeKernel<sycl::item<1, true>, 1, (lambda at /usr/include/oneapi/dpl/pstl/hetero/dpcpp/parallel_backend_sycl.h:1578:83)>>' requested here
+      1652 |     detail::CheckDeviceCopyable<KernelType>();
+           |     ^
+     /usr/bin/compiler/../../include/sycl/handler.hpp:1694:5: note: in instantiation of function template specialization 'sycl::handler::unpack<sycl::detail::RoundedRangeKernel<sycl::item<1, true>, 1, (lambda at /usr/include/oneapi/dpl/pstl/hetero/dpcpp/parallel_backend_sycl.h:1578:83)>, sycl::detail::RoundedRangeKernel<sycl::item<1, true>, 1, (lambda at /usr/include/oneapi/dpl/pstl/hetero/dpcpp/parallel_backend_sycl.h:1578:83)>, sycl::ext::oneapi::experimental::properties<std::tuple<>>, false, (lambda at /usr/bin/compiler/../../include/sycl/handler.hpp:1697:21)>' requested here
+      1694 |     unpack<KernelName, KernelType, PropertiesT,
+           |     ^
+     /usr/bin/compiler/../../include/sycl/handler.hpp:1293:7: note: in instantiation of function template specialization 'sycl::handler::kernel_parallel_for_wrapper<sycl::detail::RoundedRangeKernel<sycl::item<1, true>, 1, (lambda at /usr/include/oneapi/dpl/pstl/hetero/dpcpp/parallel_backend_sycl.h:1578:83)>, sycl::item<1, true>, sycl::detail::RoundedRangeKernel<sycl::item<1, true>, 1, (lambda at /usr/include/oneapi/dpl/pstl/hetero/dpcpp/parallel_backend_sycl.h:1578:83)>, sycl::ext::oneapi::experimental::properties<std::tuple<>>>' requested here
+      1293 |       kernel_parallel_for_wrapper<KName, TransformedArgType, decltype(Wrapper),
+           |       ^
+     /usr/bin/compiler/../../include/sycl/handler.hpp:2332:5: note: (skipping 7 contexts in backtrace; use -ftemplate-backtrace-limit=0 to see all)
+      2332 |     parallel_for_lambda_impl<KernelName, KernelType, 1, PropertiesT>(
+           |     ^
+     [...]
```

Comment:
Please add a Functor definition followed by a `sycl::is_device_copyable` traits specialization to showcase what the user is expected to do.

Reply:
5f4bda354aee113e32358df35a81802bd3666eee

Reply:
I meant also adding a definition for `MyComparator` and showing the actual `Kokkos::sort` invocation.
Do you think it is too obvious?

Reply:
Oh I thought you'd transform the lambda into that comparator thing.
I suppose that is good enough.

Reply:
The error you posted was with a lambda though right?  (Not necessarily asking you to update)

Reply:
Yes, it was with a lambda but I printed only those errors that are independent of the example. The lambda referenced in it comes from `oneDPL`.